Scripts/FilterByList/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+regex expressions are now completable with the automation
```

Comment:
please change it to
`regular expressions are now completable with the automation